db/migrate/20230610014807_create_comments.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.integer :author_id
+      t.integer :post_id
+      t.text :text
+
+      t.timestamps
+    end
+
+    add_foreign_key :comments, :users, column: :author_id
+    add_foreign_key :comments, :posts, column: :post_id
```

Comment:
- [OPTIONAL] The same here. You may also create the associations or foreign keys column using [references](https://edgeguides.rubyonrails.org/active_record_migrations.html#creating-associations-using-references) column type also available as `belongs_to`. Your code may look similar to this:

```ruby
  create_table :comments do |t|
    t.text :text
    t.timestamps
    t.references :author, foreign_key: { to_table: 'users' } # this will create author_id and a corresponding index 
    t.references :post, foreign_key: true # this will create post_id and a corresponding index
  end
```

- Note: Overall, the two code snippets achieve the same result, with the second code snippet utilizing a more concise and Rails-like syntax.